app/assets/javascripts/backbone/views/project_step_view.coffee

```diff
@@ -9,7 +9,9 @@ class MS.Views.ProjectStepView extends Backbone.View
     @initTypeSelect()
     @persisted = params.persisted
     @duplicate = params.duplicate
-    @context = @$el.data('context')
+    @timelineTableView = params.timelineTableView
```

Comment:
I don't think Fuzzy was aware but this class is for the old timeline and I want to remove it once we are fully moved over. So best not to add code to here.